django/conf/global_settings.py

```diff
@@ -390,7 +390,9 @@
 # * Note that these format strings are different from the ones to display dates
 DATETIME_INPUT_FORMATS = [
     '%Y-%m-%d %H:%M:%S',     # '2006-10-25 14:30:59'
+    '%Y-%m-%dT%H:%M:%S',     # '2006-10-25T14:30:59'
```

Comment:
I'm unsure how this change will work with localization. For example, en_GB and en_AU define their own DATETIME_INPUT_FORMATS so it seems like this change would be ignored if localization with those locales is enabled?